osf/models/user.py

```diff
@@ -1816,15 +1815,15 @@ def get_or_create_cookie(self, secret=None):
             '-expired_date'
         ).first()
 
-        if not session_store.exists(session_key=user_session_map.session_key):
-            user_session = session_store.create()
+        if not SessionStore().exists(session_key=user_session_map.session_key):
+            user_session = SessionStore()
```

Comment:
For a new session, should we still call `user_session.create()` before using it?

Reply:
Yeah we still need to `.create()`. I might have missed it here.